pkg/controller/eventing/subscription/reconcile.go

```diff
@@ -178,7 +194,7 @@ func (r *reconciler) resolveCall(namespace string, callable v1alpha1.Callable) (
 	//t := duckv1alpha1.Target{}
```

Comment:
not your doing (mine), but I neglected to to remove this TODO even though I already did this. Could you remove L193,194.

Reply:
Done.